taskcluster/fpm_taskgraph/util.py

```diff
@@ -0,0 +1,9 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Copied from https://github.com/mozilla-mobile/fenix/blob/d918cfdf3841a17545053dca2e7480c648ab4f95/taskcluster/fenix_taskgraph/util.py. I just removed the unused import.